recipes/dlib/win_boost.patch

```diff
@@ -0,0 +1,44 @@
+--- dlib/add_python_module
++++ dlib/add_python_module
+@@ -34,37 +34,16 @@
+ get_filename_component(PYTHON_PATH ${PYTHON_PATH} PATH)
+ set(CMAKE_PREFIX_PATH ${PYTHON_PATH})
+ 
+-
+-# To avoid dll hell, always link everything statically when compiling in
+-# visual studio.  This way, the resulting library won't depend on a bunch
+-# of other dll files and can be safely copied to someone elese's computer
+-# and expected to run.
+-if (MSVC)
```

Comment:
This is a choice @davisking kind made that we don't want to replicate. Other option would be to add extra flags to disable/enable this through user choice.